source/ShipInfoDisplay.cpp

```diff
@@ -163,6 +163,7 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 	attributeValues.push_back(string());
 	attributesHeight += 10;
 	double shieldRegen = attributes.Get("shield generation")
+		+ 0.5 * (attributes.Get("hot shield generation") + attributes.Get("cold shield generation"))
```

Comment:
Idk about showing it this way, but temporarily it can work ig, just should be properly expanded in the shipyard eventually, like in the other PR that adds a variable for that
Just smth to keep in mind

Reply:
I think the best way to do it would probably be to display it similarly to how acceleration and turn are shown - in this case, if there's no difference between the hot and cold performance then just show one number, but if they're different then show the limits, so for example `4000(0-6000/s)` for the bulk freighter in the test instead of 
![image](https://user-images.githubusercontent.com/18634983/218983371-e6e14c67-18e4-41ae-810a-5f51b81cac7d.png)

Reply:
That makes a lot of sense, yes!